Paintroid/src/main/res/layout/dialog_pocketpaint_text_tool.xml

```diff
@@ -60,7 +61,27 @@
                 android:id="@+id/pocketpaint_text_tool_dialog_spinner_font"
                 android:layout_width="0dp"
                 android:layout_height="match_parent"
-                android:layout_weight="3.2"/>
+                android:layout_weight="2.0"/>
+
+            <Spinner
+                android:id="@+id/pocketpaint_text_tool_dialog_spinner_text_size"
+                android:layout_weight="2.0"
```

Comment:
Change the layout weight to 2.5 and 1.5, on some displays 2.0 and 2.0 cause "Monospace" to look like 
"Monos-
 pace"

Reply:
alternatively, throw away the layout_weights and set the width to wrap_content to avoid multiline text.

Reply:
I have changed layout weight, please check again in order that I can squash commits. I tried also with wrap_content instead of layout_weights, but then layout look is not appropriate (spinners will be moved in the middle of layout).